### ISSUE TITLE
Label profile picture for VoiceOver users.

### ISSUE THREAD

**Issue by sempruijs** [element-hq/element-x-ios#2562]
### Steps to reproduce

Go to the group details page.
Go to the timeline of a group.
click on the name of the group.
This opens a details page for a group.

### Outcome

The profile picture button has no label. 
VoiceOver users hear: ```button```

I think it is important that every button has a label, even when it shows something visual. 
Please give this button a good label such as: ```profile picture```

### Your phone model

iPhone 13 Pro

### Operating system version

17.3.1

### Application version

1.5.10 (94)

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by Piciok** [comment]
Hello and my apologies for hijacking this issue but I believe it would be better to flag related issues of other unlabelled buttons in other places of the app in one place.
- The button on the main screen to start a new chat where all the recent chats are listed (the last element in the focus order);
- The button to close the above mentioned dialogue;
- A button in the chat window, between the edit field for a new message and the voice message recording, most likely an Attach Multimedia button, except when tapping on it, nothing happens that I can notice;
I hope those can be fixed.
